test/manual/mocharc.json

```diff
@@ -1,3 +1,4 @@
 {
-  "require": "ts-node/register"
+  "require": "ts-node/register",
+  "reporter": "spec-xunit-file"
```

Comment:
hm this is specified in the script above, can we reduce it to one place?

Reply:
I didn't add `--report spec-xunit-file` to the npm scripts that use this file via the `--config` option in order to reduce verbosity. I can expand it out into the package scripts if desired.

Reply:
Can't you add this same setting to `.mocharc.json`?